src/python/pants/backend/docker/util_rules/docker_binary.py

```diff
@@ -34,6 +34,7 @@ def build_image(
         digest: Digest,
         dockerfile: str | None = None,
         build_args: DockerBuildArgs | None = None,
+        context_root: str = ".",
```

Comment:
Does it make sense for these to all be default args? A risk with defaults is that call sites forget to set them.

Reply:
Maybe not all.. I'll review what makes sense, and drop the defaults for those that are usually overridden.